doc/dev/jax_integration.rst

```diff
@@ -133,6 +133,11 @@ array([[1., 1., 1., 1.],
        [1., 1., 1., 1.],
        [1., 1., 1., 1.]])
 
+Dynamic arrays can be created using ``jnp.ones``, ``jnp.zeros``, however note that ``jnp.arange``
+and ``jnp.linspace`` do not currently support generating dynamically-shaped arrays (however, unlike
```

Comment:
two howevers in one sentence? 😅

Reply:
Yeah that also annoyed me, but I was banking on no-one noticing 😅 

let me remove the first however.